.rubocop.yml

```diff
@@ -6,8 +6,8 @@ AllCops:
     - 'bin/**'
     - 'db/schema.rb'
     - 'vendor/**'
-  TargetRubyVersion: 2.3
-  TargetRailsVersion: 5.1
+  TargetRubyVersion: 2.6
```

Comment:
Only useful if the target machine is also running Ruby 2.6, is that confirmed?

Reply:
It does. I've upgraded a few weeks ago including upgrade to buster.

Reply:
Noice :)